electron/main/Files/Filesystem.ts

```diff
@@ -116,10 +116,15 @@ export function createFileRecursive(
   charset?: BufferEncoding
 ): void {
   const dirname = path.dirname(filePath);
+
   if (!fs.existsSync(dirname)) {
     fs.mkdirSync(dirname, { recursive: true });
   }
 
+  if (fs.existsSync(filePath)) {
+    return;
+  }
+
   fs.writeFileSync(filePath, content, charset);
```

Comment:
This line looks like dead code now that the prior 2 covers all possible scenarios

Reply:
Wdym?

Reply:
sorry I misread it...I thought the existsSync was being applied to the same `dirname`